Changelog.md

```diff
@@ -1,3 +1,10 @@
+## 2.4.3 - unreleased
+
+### Fixed
+
+-   Issues with the UI layout in the **Certificate Manager** tab.
+-   Issue with editing the security tag in the **Certificate Manager** tab, which would cause the form to freeze when a non-numeric value was passed.
```

Comment:
```suggestion
-   Issue with editing the security tag in the **Certificate Manager** tab, which would cause the input to become uneditable when a non-numeric value was passed.
```